_scripts/instruction-widget/templates/install/commonauto.html

```diff
@@ -2,7 +2,9 @@
 Since it doesn't seem like your operating system has a packaged version of Certbot, you should use our certbot-auto script to get a copy:
 </p>
 <pre>wget https://dl.eff.org/certbot-auto
-chmod a+x certbot-auto</pre>
+sudo mv certbot-auto /usr/local/bin/certbot-auto
+sudo chown root:root /usr/local/bin/certbot-auto
```

Comment:
I think we should use `chown 0:0` here. `chown 0:0` looks a little odd and I've personally never seen it used before, however, there's not a `root` group on the BSD systems I looked at. Instead, there's a `wheel` group with GID 0.

Reply:
I'm afraid that this might confuse people, as I haven't seen that used either. I'd propose to instead `chown root /usr/local/bin/certbot-auto`, and to `chmod 0755 ...` (as a solution to other comment as well). That ensures that the group wouldn't have the write permissions anyway.

Reply:
Works for me!